guides/common/modules/proc_viewing-the-project-ansible-modules.adoc

```diff
@@ -1,6 +1,11 @@
 [id="Viewing_the_Ansible_Modules_{context}"]
 = Viewing the {Project} Ansible Modules
 
+:link-desc: Ansible Galaxy Portal
+ifdef::satellite[]
+:link-desc: {RHCloud}
+endif::[]
```

Comment:
shouldn't this be in `attributes`?

Reply:
To avoid sentence duplication for upstream and downstream, I kept that attribute limited to this particular module only. However, I am open to a better way.

Reply:
Where you keep the attribute doesn't affect the sentence at all.

Reply:
`:link-desc:` working as desired after that placement.

Reply:
Please rename it to something like "ansible-galaxy-name" and place it in "attributes-base.adoc".

Reply:
Addressed